docker-compose.yml

```diff
@@ -3,13 +3,17 @@ version: '3'
 services:
   web:
     image: martel/mastermind-ui
+    environment:
+      - MASTERMIND_API_URI=http://localhost:3000
```

Comment:
@chicco785 Since the UI is a Vue.js app and I'm using axios to interact with the API I can't use Docker's service discovery directly, because these API calls happen from the web browser, not from the application itself running in the UI container, so "api" won't resolve to the API's host. I'll try figuring out a better way to do this at some point, because it is a bit inconvenient.